crates/sui-sdk/src/lib.rs

```diff
@@ -53,21 +129,46 @@ impl Default for SuiClientBuilder {
 }
 
 impl SuiClientBuilder {
+    /// Set the request timeout to the specified duration
     pub fn request_timeout(mut self, request_timeout: Duration) -> Self {
         self.request_timeout = request_timeout;
         self
     }
 
+    /// Set the max concurrent requests allowed
     pub fn max_concurrent_requests(mut self, max_concurrent_requests: usize) -> Self {
         self.max_concurrent_requests = max_concurrent_requests;
         self
     }
 
+    /// Set the WebSocket URL for the Sui network
     pub fn ws_url(mut self, url: impl AsRef<str>) -> Self {
         self.ws_url = Some(url.as_ref().to_string());
         self
     }
 
+    /// Returns a [`SuiClient`] object connected to the Sui network running at the URI provided
```

Comment:
nit: Does this render correctly if you put backticks in the link?  I see that you tend not to do that elsewhere -- we should be consistent.